src/apps/EDGE3D.hpp

```diff
@@ -360,6 +360,196 @@ RAJA_INLINE void edge_MpSmatrix(
   }
 }
 
+RAJA_HOST_DEVICE
+RAJA_INLINE void symmetric_edge_MpSmatrix(
+  const rajaperf::Real_type  (&x)[NB],
+  const rajaperf::Real_type  (&y)[NB],
+  const rajaperf::Real_type  (&z)[NB],
+  rajaperf::Real_type        alpha,
+  rajaperf::Real_type        beta,
+  const rajaperf::Real_type  detj_tol,
+  const rajaperf::Int_type   quad_type,
+  const rajaperf::Int_type   quad_order,
+  rajaperf::Real_type        (&matrix)[EB][EB])
```

Comment:
There is still a full matrix?

Reply:
The interface that this is used for requires having a full matrix. However, when we we are computing the work at each quadrature point, we are using a symmetric matrix.

Reply:
I can follow up with a study on how beneficial it would be to never create the full matrix. If that is a major impact, that may be enough incentive to rewrite how things are done in the ultimate use case.

Reply:
Did  a quick study measuring the performance of an impl that has no 12x12 matrices. Its indistinguishable from the performance seen in this MR.